screenpy/pacing.py

```diff
@@ -3,17 +3,27 @@
 (cases), and provide the gravitas (severity) of those groupings. These will
 run through all of the Narrator's adapters.
 """
-
 import re
 from functools import wraps
 from typing import Any, Callable, Optional
 
 from screenpy.narration import Narrator, StdOutAdapter
 
 Function = Callable[..., Any]
+
 the_narrator: Narrator = Narrator(adapters=[StdOutAdapter()])
 
 
+def function_should_log_none(func: Function) -> bool:
```

Comment:
Should this be a private function? `_function_should_log_none`?

I'm guessing someone somewhere would probably like to use it, but we might want to signal that it's purpose is very narrow. Or maybe we don't? 🤔

Reply:
I'm not worried about anyone using it since it's not in the main package `__init__`.  If they want to import it and use it... shouldn't be an issue for us.